.gitignore

```diff
@@ -11,14 +11,16 @@
 .mtj.tmp/
 
 # Package Files #
-*.jar
+# *.jar
```

Comment:
Shall we completely remove this rather than commenting?

Reply:
+1